_observing-your-data/query-insights/query-metrics.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Query metrics
+parent: Query insights
+nav_order: 20
+---
+
+# Query metrics
+
+Query metrics involves comprehensive instrumentation along the search path to capture key [metrics](#metrics), such as aggregation types, query types, latency, and resource usage per query type. Open Telemetry (oTel) is the instrumentation framework. The telemetry data can be consumed using oTel metrics [exporters]({{site.url}}{{site.baseurl}}/observing-your-data/trace/distributed-tracing/#exporters).
+
+## Configuring query metrics
+
+To configure query metrics, use the following steps.
+
+### Step 1: Install the Query Insights plugin
+
+For information about installing the Query Insights plugin, see [Installing the Query Insights plugin]({{site.url}}{{site.baseurl}}/observing-your-data/query-insights/index/#installing-the-query-insights-plugin).
+
+### Step 2: Install the OpenTelemetry plugin
+
+For information about installing the OpenTelemetry plugin, see [Distributed tracing]({{site.url}}{{site.baseurl}}/observing-your-data/trace/distributed-tracing/).
+
+### Step 3: Enable query metrics
+
+Enable query metrics by configuring the following `opensearch.yml` settings:
+
+```yaml
+telemetry.feature.metrics.enabled: true
+search.query.metrics.enabled: true
+```
+{% include copy.html %}
+
+The following is a complete sample configuration that includes telemetry configuration:
+
+```yaml
+# Enable query metrics feature
+search.query.metrics.enabled: true
+telemetry.feature.metrics.enabled: true
+
+# oTel-related confuration
+opensearch.experimental.feature.telemetry.enabled: true
+telemetry.tracer.sampler.probability: 1.0
+telemetry.feature.tracer.enabled: true
+```
+{% include copy.html %}
+
+Alternatively, you can configure query metrics using the API:
+
+```json
+PUT _cluster/settings
+{
+  "persistent" : {
+    "search.query.metrics.enabled" : true
+  }
+}
+```
+{% include copy-curl.html %}
+
+Configure the export of metrics and traces using a gRPC exporter. For more information, see [Exporters]({{site.url}}{{site.baseurl}}/observing-your-data/trace/distributed-tracing/#exporters). You can skip this step if you use the [default logging exporter](#default-logging-exporter):
+
+```yaml
+telemetry.otel.tracer.span.exporter.class: io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter
+telemetry.otel.metrics.exporter.class: io.opentelemetry.exporter.otlp.metrics.OtlpGrpcMetricExporter
+```
+{% include copy.html %}
+
+## Metrics
+
+Query metrics support the following measurements:
+
+- The number of queries per query type (for example, the number of `match` or `regex` queries).
```

Comment:
These technically don't need periods because they aren't complete sentences and don't complete the introductory stem.